modules/codegen/src/main/scala/com/twilio/guardrail/protocol/terms/server/ServerTerms.scala

```diff
@@ -63,7 +68,7 @@ abstract class ServerTerms[L <: LA, F[_]] {
           List[StrictProtocolElems[L]],
           Map[String, SecurityScheme[L]]
       ) => F[RenderedRoutes[L]] = generateRoutes _,
-      newGetExtraRouteParams: Boolean => F[List[L#MethodParameter]] = getExtraRouteParams _,
+      newGetExtraRouteParams: (Boolean, Boolean) => F[List[L#MethodParameter]] = getExtraRouteParams _,
```

Comment:
this highlights the danger of permitting primitive types in these `copy` helpers